charts/ingress-nginx/templates/controller-service-internal.yaml

```diff
@@ -14,7 +14,7 @@ metadata:
   {{- end }}
   name: {{ include "ingress-nginx.controller.fullname" . }}-internal
```

Comment:
Maybe once the changes recommended by @aledbf are complete, this one can be approved!